src/Codeception/Module/Db.php

```diff
@@ -139,10 +193,11 @@ class Db extends CodeceptionModule implements DbInterface
      * @var array
      */
     protected $config = [
-        'populate' => true,
-        'cleanup' => true,
+        'populate' => false,
+        'cleanup' => false,
```

Comment:
Yes. They are dangerous as they are now. It took me few years to conclude that. 
I should not forget to add this in changelog.
This may break some BC but changing config values shoudn't be a big deal...